numpy/_core/code_generators/ufunc_docstrings.py

```diff
@@ -2860,6 +2860,54 @@ def add_newdoc(place, name, doc):
     .. versionadded:: 1.10.0
     """)
 
+add_newdoc('numpy._core.umath', 'vecdot',
+    """
+    Vector dot product of two arrays.
+
+    Let :math:`\\mathbf{a}` be a vector in `x1` and :math:`\\mathbf{b}` be
+    a corresponding vector in `x2`. The dot product is defined as:
+
+    .. math::
+       \\mathbf{a} \\cdot \\mathbf{b} = \\sum_{i=0}^{n-1} \\overline{a_i}b_i
+
+    where the sum is over the last dimension (unless `axis` is specified) and
+    where :math:`\\overline{a_i}` denotes the complex conjugate if :math:`a_i`
+    is complex and the identity otherwise.
+
+    Parameters
+    ----------
+    x1, x2 : array_like
+        Input arrays, scalars not allowed.
+    out : ndarray, optional
+        A location into which the result is stored. If provided, it must have
+        a shape that the broadcasted shape of `x1` and `x2` with the last axis
+        removed. If not provided or None, a freshly-allocated array is used.
+    **kwargs
+        For other keyword-only arguments, see the
+        :ref:`ufunc docs <ufuncs.kwargs>`.
+
+    Returns
+    -------
+    y : ndarray
+        The vector dot product of the inputs.
+        This is a scalar only when both x1, x2 are 1-d vectors.
+
+    Raises
+    ------
+    ValueError
+        If the last dimension of `x1` is not the same size as
+        the last dimension of `x2`.
+
+        If a scalar value is passed in.
+
+    See Also
+    --------
+    vdot : same but flattens arguments first
+    einsum : Einstein summation convention.
+
+    .. versionadded:: 2.0.0
```

Comment:
For some reason this formatting is crashing the docs build:

```
sphinx.errors.ExtensionError: Handler <function mangle_signature at 0x7f2c3274c220> for event 'autodoc-process-signature' threw an exception (exception: Error parsing See Also entry '.. versionadded:: 2.0.0' in the docstring of vecdot)
```

Maybe something to do with it appearing at the end of the docstring? Other usages in the codebase seem to place this earlier in the docstring.

Reply:
that's super weird, the function just above, `matmul`, has a line at the end of the docstring too.  Maybe it has to do with being at the end of `See Also`.  Maybe it is a good reason to add a few examples...

Reply:
OK, I added a single example. Hopefully that fixes the failure...

Reply:
The docs build now passed, so I think it likely had something to do with being in the `See Also` section. Anyway, for this PR the problem is solved...